Gemfile

```diff
@@ -3,27 +3,17 @@ source 'https://rubygems.org'
 gem 'kafo', '~> 4.1'
 gem 'librarian-puppet'
 gem 'puppet', ENV.key?('PUPPET_VERSION') ? "~> #{ENV['PUPPET_VERSION']}" : '~> 6.0'
-if RUBY_VERSION < '2.1'
-  gem 'puppet-strings', '< 2.0.0'
-else
-  gem 'puppet-strings'
-end
+gem 'facter', '~> 4.0'
+
+gem 'puppet-strings'
 gem 'rake'
-gem 'rdoc', '< 6' if RUBY_VERSION < '2.2'
 
 group :test do
   gem 'rspec'
   gem 'rubocop', '~> 0.50.0'
 end
 
 group :development do
-  # Needed to pin dependencies
```

Comment:
This comment is still relevant: they're here for `rake pin_modules`. I agree pinning can be a bit overloaded in this context.